pynecone/components/media/icon.py

```diff
@@ -36,5 +38,30 @@ def create(cls, *children, **props):
             )
         if "tag" not in props.keys():
             raise AttributeError("Missing 'tag' keyword-argument for Icon")
+        else:
+            tag_keywords = [
```

Comment:
May use tuple instead of list to avoid change this by accident?

Reply:
I'm a bit confused - what are the `tag_keywords` used for? Shouldn't the `.title()` call be enough?

Reply:
No, that isn't enough for "multiword" tags (eg. `ExternalLinkIcon`, `RepeatClockIcon`). From what I found in testing, `tag` should still match actual Chakra icon name case to work

Reply:
Ah I see - maybe in those cases the user can specify the tag in `snake_case` like `pc.icon(tag="repeat_clock_icon")` and we can use the `utils.to_title_case` function in `utils.py`. What do you think?

Reply:
Yes, that sounds better and more futureproof. We just need to make it clear that the user has to use `snake_case` in documentation